sncosmo/simulation.py

```diff
@@ -128,6 +128,11 @@ def realize_lcs(observations, model, params, thresh=None,
         If True, only observations with times between
         ``model.mintime()`` and ``model.maxtime()`` are included in
         result table for each SN. Default is False.
+    scatter : bool, optional, defaults to True
```

Comment:
Nitpick: "Default is True" should go at the end of the description, not on this line. See above parameter for example.